### DISCUSSION TITLE
Reports all sites when scanning a system

### DISCUSSION THREAD

**Discussion by Bloch-Boas** [EDCD/EDDI#2473] [EDDI VoiceAttack Q&A] [answered]
Hi guys, I do not know if this is the place... But I use EDDI with Voice Attack, and when I scan a system in FSS mode, it reports all conflict zones, extraction sites, ect. ect....

I have tried to look in "Signal Detected" and only the things I would like to get told are set to "true"...  (it is basically the default file)
but I still get all the sites listed... is it the wrong file I look in or does Voíce Attack just ignore settings?

So basically what I am looking for, is the settings file for what to announce in FSS mode... I guess...

Best regards
Andreas

**Comment by Bloch-Boas** [comment, marked as answer]
The true/false statements in the Signal detected are typically applied only to the signals detected when EDDI enters a system, not to signals detected when in fss mode or in saa mode. If you'd like to apply them in fss mode then you can change lines 33-34 from

`{if status.gui_focus = "fss mode" || status.gui_focus = "saa mode" || (spokenSources[event.source] = true && event.unique):`

to
`{if status.gui_focus = "saa mode" || (spokenSources[event.source] = true && event.unique):`


Alternatively, you can use

`{if spokenSources[event.source] = true && event.unique):`

if you'd like to also filter the signals detected while in saa mode and launching probes at the surface of a body.
